koku/api/user_settings/views.py

```diff
@@ -29,6 +39,47 @@ def __init__(self, message):
         self.detail = {"detail": force_text(message)}
 
 
+@dataclass
+class SettingParamsHandler:
+    setting: str
+    retriever: object = field(init=False)
+    update_param: object = field(init=False)
+
+    def __post_init__(self):
+        if self.setting in ["cost_type", "cost-type"]:
```

Comment:
`cost_type` was not valid before, should we make it valid now?

Reply:
I can remove it. 👍